recipes/torchio/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "torchio" %}
+{% set version = "0.18.29" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchio-{{ version }}.tar.gz
```

Comment:
There's one last "torchio" here that should be `{{ name }}`

Reply:
Done!